Metric/Metric.php

```diff
@@ -198,4 +198,14 @@ private function resolveTagValue(string $valueToResolve, array $resolvers): ?str
                 return $valueToResolve;
         }
     }
+
+    private function correctValue(string $value): string
+    {
+        /* @see https://github.com/prometheus/statsd_exporter/pull/178/files#diff-557eb2a359922e8de5f18397fed0cd99R423 */
+        if ($this->getResolvedType() === self::STATSD_TYPE_TIMER) {
+            return $value * 1000;
```

Comment:
the console event above is already in microseconds

Reply:
Both the Console and Kernel monitoring evens are indeed in seconds.